cf/s3.yaml

```diff
@@ -113,3 +113,38 @@ Resources:
             AllowedOrigins:
               - '*'
             Id: uiHeadersRule
+  PublicDatasetsBucket:
+      Type: AWS::S3::Bucket
+      Properties:
+        BucketName: !Sub "biomage-public-datasets-${Environment}"
+        PublicAccessBlockConfiguration:
+          BlockPublicAcls: True
+          BlockPublicPolicy: True
+          IgnorePublicAcls: True
+          RestrictPublicBuckets: True
+        VersioningConfiguration:
+          Status: "Enabled"
+        LifecycleConfiguration:
+          Rules:
+            # Non-current versions of source files are stored in normal S3 for
+            # 5 days and are then deleted.
+            - Id: PreserveThenDeleteNonCurrentVersion
+              Status: "Enabled"
+              NoncurrentVersionExpirationInDays: 5
+        CorsConfiguration:
+          CorsRules:
+            - AllowedHeaders:
+                - '*'
+              ExposedHeaders:
+                - ETag
+                - "x-amz-server-side-encryption"
+                - "x-amz-request-id"
+                - "x-amz-id-2"
+              AllowedMethods:
+                - GET
+                - PUT
+                - POST
+                - DELETE
+              AllowedOrigins:
+                - '*'
+              Id: uiHeadersRule
```

Comment:
I had to add this part for the upload to work in `biomage-originals`. Since we are not doing that in your new bucket, do you know if it fails if you remove this?